models/marts/ledger_current_state/ttl_current.yml

```diff
@@ -0,0 +1,78 @@
+version: 2
+
+models:
+  - name: ttl_current
+    description: '{{ doc("ttl_current") }}'
+    tests:
+      - dbt_utils.recency:
+          datepart: hour
```

Comment:
does this truncate the timestamp to the hourly level? Like if data was older than 24+ hours, would we get an alert? or is this truly evaluating only the hour part of the datetime field?

Reply:
The truncate should be fine because datetime still has the hh:mm:ss part of the timestamp. I'm not sure about the 24+ hours alert. I would hope that dbt_utils is smart enough to take the datetime_diff instead of the extract(hour) - extract(hour)

We should probably test it but also we would get an error before this would be a problem because the _current table are run every 30 mins so we would hit the 12 hour recency test regardless

Reply:
sounds good. let's test and adjust appropriately